docs/sources/alerting/fundamentals/data-source-alerting.md

```diff
@@ -0,0 +1,39 @@
+---
+aliases:
+  - /docs/grafana/latest/alerting/fundamentals/data-source-alerting/
+description: Data sources in Grafana Alerting
+title: Data sources
+weight: 100
+---
+
+# Data sources
+
+There are a number of data sources that are compatible with Grafana Alerting. Each data source is supported by a plugin. You can use one of the built-in data sources listed below, use [external data source plugins](https://grafana.com/grafana/plugins/?type=datasource), or create your own data source.
+
+If you are creating your own data source, make sure it is a backend plugin as Grafana Alerting requires this in order to be able to evaluate rules using the data source. Frontend data sources are not supported, because the evaluation engine runs on the backend.
```

Comment:
```suggestion
If you are creating your own data source plugin, make sure it is a backend plugin as Grafana Alerting requires this in order to be able to evaluate rules using the data source. Frontend data sources are not supported, because the evaluation engine runs on the backend.
```